<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;Gotham Pro Black&quot;;
margin:0;</string>
  </property>
  <widget class="QWidget" name="centralwidget501">
   <widget class="QPushButton" name="pushButton501">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>210</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_502">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_503">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>310</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_504">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_505">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>510</y>
      <width>189</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_506">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_507">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_508">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_509">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_512">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>410</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_511">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_513">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>510</y>
      <width>189</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_514">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_515">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_510">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>128</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit501">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_502">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>511</width>
      <height>106</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar500">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
